src/packages/dca/api/dcaApiCasting.ts

```diff
@@ -177,13 +181,13 @@ export class DcaApiCasting {
 						? totalBought / BigInt(tradesPrepared.length)
 						: BigInt(0),
 				totalSpent: totalSpent,
-				interval: BigInt(response.frequency_ms),
+				intervalMs: BigInt(response.frequency_ms),
```

Comment:
i mentioned before please see old comment but this should be a number not bigint